### ISSUE TITLE
XCode 12.2 and swift 5 iOS 14 problem 

### ISSUE THREAD

**Issue by xhidnoda** [Aiur3908/AYPopupPickerView#6]
I had this error when try to open a UIPickerDateView:
**AYPopupPickerView show a white empty screen** with just one date and two options (Cancel and OK) head buttons. 

Whet tap the AYPopupPickerView this is appears in the log:
**[Assert] Failed to find a presenting view controller for view (<_UIDatePickerIOSCompactView: 0x7fc7b7c06980; frame = (0 0; 390 770); gestureRecognizers = <NSArray: 0x60000253be40>; layer = <CALayer: 0x600002b1f3c0>>) in window (<UIWindow: 0x7fc7b7c0f5e0; frame = (0 0; 390 844); autoresize = W+H; gestureRecognizers = <NSArray: 0x60000256eb80>; layer = <UIWindowLayer: 0x600002b76540>>). The interaction's view (or an ancestor) must have an associated view controller for presentation to work.**

This only happens in XCode 12.2  swift 5 and iOS 14. Previus version is work just perfect!

Can anyone help me?

**Comment by Aiur3908** [comment]
Because iOS 13.4+ UIDatePickerView have new style.
I fixed it in version 1.2. 
You can use `pod update` to update newer version.

**Comment by xhidnoda** [comment]
Yeah... i try to update whit `pod update` but my pod has no update anything, i'm don't why....but i just fixes whit this code: 
```
if #available(iOS 14.0, *) {
            popupDatePickerViewNacimiento.datePickerView.preferredDatePickerStyle = .wheels
        } else {
            // Fallback on earlier versions
        }
```

Thanks @Aiur3908 !!

**Comment by Aiur3908** [comment]
You can use `pod cache clean AYPopupPickerView` to clean CocoaPods cache and then try `pod update`
